plugins/module_utils/gitlab.py

```diff
@@ -74,7 +74,10 @@ def gitlab_authentication(module):
     gitlab_job_token = module.params['api_job_token']
 
     if not HAS_GITLAB_PACKAGE:
```

Comment:
How about moving that to a function which is called from here (for backwards compatibility), and also from all the places in the modules where they did the check themselves?

Reply:
All the modules call `gitlab_authentication()`. Wouldn't that be enough?

Reply:
Well, some use `gitlab` **before** they call that function.

Reply:
... and I just read it through. I see where you're going with this. Yup, fixing it.

Reply:
done